src/api/java/mezz/jei/api/IModPlugin.java

```diff
@@ -1,16 +1,21 @@
 package mezz.jei.api;
 
+import java.util.Comparator;
+
 import mezz.jei.api.ingredients.IModIngredientRegistration;
 import mezz.jei.api.recipe.IRecipeCategoryRegistration;
+import net.minecraft.item.ItemStack;
 
 /**
- * The main class to implement to create a JEI plugin. Everything communicated between a mod and JEI is through this class.
- * IModPlugins must have the {@link JEIPlugin} annotation to get loaded by JEI.
+ * The main class to implement to create a JEI plugin. Everything communicated
```

Comment:
Please do not reformat existing lines to the wrong format.
???

Reply:
This is a side-effect of using ctrl-shift-F in eclipse to fix the spaces to tabs issue.  I did not anticipate this.  How do you ensure indenting is all tabs and not spaces?  I'll go and try to undo as many of these as I can find.